### ISSUE TITLE
Promote from hca/dev all the way to hca/prod

### ISSUE THREAD

**Issue by hannes-ucsc** [DataBiosphere/azul#1902]
- [x] Check if `hca/dev` was reindexed during most recent deployment. If not, reindex `hca/dev`.
- [x] Promote to `hca/integration` and reindex
- [x] Promote to `hca/staging` and reindex
- [x] Smoke test Data Browser instance for `hca/staging`
- [x] Prepare relesae notes; announce on Slack `#dcp-ops` channel
- [x] Check in with @theathorn and @hannes-ucsc 
- [x] Promote to `hca/prod` and reindex
- [x] Smoke test Data Browser instance for `hca/prod`

This mostly to get #1853 out the door.

cc: @kbergin & @jessicaway

**Comment by melainalegaspi** [comment]
Wait for fix to #1772.

**Comment by melainalegaspi** [comment]
Removed #1903 as blocker and ready to proceed after #1853 has been demoed today.

**Comment by melainalegaspi** [comment]
@achave11 will create new ticket to finish new backports of fixes to issues that occurred during this promotion, closing this one
